From 8935560c38e5beb4d3fd295622703308479693ca Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Adam=20S=C5=82abo=C5=84?= <asaillen@protonmail.com>
Date: Mon, 1 Jul 2024 12:51:36 +0200
Subject: [PATCH] Wine workarounds

---
 configure.cmd | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure.cmd b/configure.cmd
index d94501ab4bc..b9cbaf90b85 100755
--- a/configure.cmd
+++ b/configure.cmd
@@ -52,10 +52,10 @@ if defined ROS_ARCH (
     cl 2>&1 | find "ARM" > NUL && set ARCH=arm
     cl 2>&1 | find "ARM64" > NUL && set ARCH=arm64
     cl 2>&1 | find "19.00." > NUL && set VS_VERSION=14
-    cl 2>&1 | findstr /R /c:"19\.1.\." > NUL && set VS_VERSION=15
-    cl 2>&1 | findstr /R /c:"19\.2.\." > NUL && set VS_VERSION=16
-    cl 2>&1 | findstr /R /c:"19\.3.\." > NUL && set VS_VERSION=17
-    cl 2>&1 | findstr /R /c:"19\.4.\." > NUL && set VS_VERSION=17
+    cl 2>&1 | find "19.1" > NUL && set VS_VERSION=15
+    cl 2>&1 | find "19.2" > NUL && set VS_VERSION=16
+    cl 2>&1 | find "19.3" > NUL && set VS_VERSION=17
+    cl 2>&1 | find "19.4" > NUL && set VS_VERSION=17
     if not defined VS_VERSION (
         echo Error: Visual Studio version too old ^(before 14 ^(2015^)^) or version detection failed.
         goto quit
-- 
2.45.2

